### DISCUSSION TITLE
orthogonal_bitmap_expr_calculate进行交集（&）时错误

### DISCUSSION THREAD

**Discussion by mozhimin** [apache/doris#19400] [A - General / Q&A]
SELECT orthogonal_bitmap_expr_calculate(nascent_id, tag_value, '(女)&(25～35岁)')
from ads_user_profile_tag_value_user_bitmap
返回值为0；

标签值为“女”且“25~35岁”的人很多，用下面这种写法没问题，是不是说明&求交集这个方式有问题
SELECT bitmap_count(BITMAP_INTERSECT(nascent_id))
from ads_user_profile_tag_value_user_bitmap
WHERE  tag_value = '女' or tag_value = '25～35岁'

**Comment by mozhimin** [comment]
Can anyone answer it

**Reply by yagagagaga** [reply]
Do you mean `WHERE tag_value = '女' and tag_value = '25～35岁'` not `WHERE tag_value = '女' or tag_value = '25～35岁'`?

**Comment by LemonLiTree** [comment]
try: 
SELECT orthogonal_bitmap_expr_calculate(nascent_id, tag_value, '("女")|("25～35岁")')
from ads_user_profile_tag_value_user_bitmap
